contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract OETHHarvesterSimple is Governable {
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    address public strategist;
+    mapping(address => bool) public supportedStrategies;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event Harvested(address token, uint256 amount);
+    event StrategistChanged(address strategist);
+    event SupportedStrategyUpdate(address strategy, bool status);
```

Comment:
I think our usual format for events that change configuration is to use "Updated" at the end of the event name.

<img width="617" alt="image" src="https://github.com/user-attachments/assets/56806c23-bc65-42ee-b6df-2b0a30c05f6e" />

Reply:
Changed in this commit: [9e968fd](https://github.com/OriginProtocol/origin-dollar/pull/2333/commits/9e968fdd6e9173e3f22a02a9d89ccfccfb53f12f)